src/pq/conninfo.cr

```diff
@@ -36,7 +36,7 @@ module PQ
     def initialize(host : String? = nil, database : String? = nil, user : String? = nil, @password : String? = nil, port : Int | String? = 5432, sslmode : String | Symbol? = nil)
       @host = default_host host
       db = default_database database
-      @database = db.starts_with?('/') ? db[1..-1] : db
```

Comment:
Why this change? It doesn't seem to be related to IPv6 hostname and changes the behaviour.

Reply:
Yes that's lchop, a mistake.
That's in an other commit.

Reply:
`db.lchop?('/') || db` == `db.chop('/')`